### DISCUSSION TITLE
Can't start in Debian Linux, period.

### DISCUSSION THREAD

**Discussion by violent-froggy00** [stakira/OpenUtau#969] [Q&A]
I've tried every solution and asked many, many people for help, but its just not going anywhere. No matter what I try, I can't run the tarball and the installer is being very troublesome as well. I'd appreciate ANY help.

**Comment by pseudo-producer** [comment]
Hello! Did you follow the installation [script made for Linux that a user created](https://hitcoder.tubs.wtf/Posts/openutau/)? For me it always works!

**Comment by pseudo-producer** [comment]
After this process, you are left with Resamplers, which only work via Wine and through an easy-to-create script, just follow [this tutorial](https://www.youtube.com/watch?v=4BSxfS3cA4o).

**Comment by pseudo-producer** [comment]
I only noticed now that you made this request on my birthday... LOL
